yacc_error6.py returned with 1

Traceback (most recent call last):
  ...
ImportError: No module named lex

